src/docs/development/ios-project-migration.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Xcode 11.4 Support
+description: How to migrate existing Flutter iOS projects to Xcode 11.4.
+---
+
+{{site.alert.note}}
+  You might be directed to this page if Flutter detects that your project
+  doesn't support Xcode 11.4.
+{{site.alert.end}}
+
+To develop Flutter apps for iOS, you need a Mac with Xcode installed.
+Xcode 11.4 changed the way frameworks are linked and embedded, and
+you may [experience issues switching between iOS devices and simulators][].
+Flutter v1.15.3 and later will automatically migrate your Xcode project.
+
+If you need to manually upgrade your project, use the following steps:
+<ol markdown="1">
+<li markdown="1">From the Flutter app directory, open `ios/Runner.xcworkspace` in Xcode.
```

Comment:
Does xcworkspace exist if the project doesn't use plugins? I don't remember.

Reply:
Yes, it's always added as part of the create template.